### ISSUE TITLE
question: Project is deprecated?

### ISSUE THREAD

**Issue by jkroepke** [aquasecurity/tfsec#2047]
Hi,

on KubeCon 2023 Europe, I got this information from an aqua employee that tfsec is deprecated and merged into trivy.

Is this true? I cant find any related informations about this.

**Comment by dupuy26** [comment]
#2020
#2021

**Comment by wyardley** [comment]
There is a banner on the top of https://github.com/aquasecurity/tfsec/discussions
I asked a similar question in #1997

I suggested that it might be good to make this more obvious in some of the places that people would be likely to look, and I _think_ there is some work being done towards that.

**Comment by GoodmanBen** [comment]
I agree that this needs to be made much more clear. At time of writing, a user can read the entire README.md file and there is no indication about the trivy migration. Our team was only aware of the migration after investigating why the last release occurred in October of 2022.

This PR looks like it will help make it more obvious by including information directly in the README: https://github.com/aquasecurity/tfsec/pull/2020
